content/en/users/aai/check-in/obtaining-tokens/oidc-agent/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: oidc-agent
+linkTitle: oidc-agent
+type: docs
+weight: 40
+aliases:
+  - /users/aai/check-in/obtaining-tokens/oidc-agent
```

Comment:
```suggestion
  - /users/aai/check-in/obtaining-tokens/oidc-agent
```

I'm confused, why are you adding this alias? Isn't it the current real path?

Reply:
I copied the header of the document from another page and adapted it for oidc-agent doc.
I have removed the alias since it's not needed.